assets/blocks/course-outline/module-block/module-edit.js

```diff
@@ -158,6 +164,7 @@ export const ModuleEdit = ( props ) => {
 							onChange={ updateName }
 						/>
 					</h2>
+					{ teacher && <span>({ teacher })</span> }
```

Comment:
Can we add tests for it?

Reply:
Added here https://github.com/Automattic/sensei/pull/5114/commits/159d01612aeb32e52f81cd6eff4dd5edd17bf19e , can you kindly take a look now?